.babelrc

```diff
@@ -14,7 +14,10 @@
         ["env", {
           "targets": {
             "node": "6.11.4",
-          }
+          },
+          "modules": "commonjs",
```

Comment:
for some reason, this is the only way I could get the tests to work?! I don't understand why it wasn't necessary for the cli.